packages/components/psammead-topic-tags/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@bbc/psammead-topic-tags",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "module": "esm/index.js",
+  "sideEffects": false,
+  "description": "Contains topic links",
```

Comment:
If these packages weren't public, this wouldn't matter as much, but can we make this a little more descriptive and relevant to people outside of our circle?